ReactAndroid/src/main/java/com/facebook/react/packagerconnection/PackagerConnectionSettings.java

```diff
@@ -54,10 +54,21 @@ public String getDebugServerHost() {
   }
 
   public void setDebugServerHost(String host) {
```

Comment:
Do we know if this class can be accessed from multiple threads? Since we are introducing a non-final class member variable, we need to make sure it's accessed through proper synchronisation if that's the case.